com.musala.simple.smarthome/com/musala/simple/smarthome/devices/SmartBulb.java

```diff
@@ -0,0 +1,12 @@
+package com.musala.simple.smarthome.devices;
+
+import com.musala.simple.smarthome.ISmartDevice;
+
```

Comment:
Add javaDoc to the class. This is valid for all classes in the `devices` package.